PDSA_System/Server/DataBase/Create.sql

```diff
@@ -10,13 +11,19 @@ CREATE TABLE IF NOT EXISTS Bruker(
   Opprettet DATETIME NOT NULL,
   CONSTRAINT PK_Bruker PRIMARY KEY(BrukerId)
 );
+INSERT INTO NordicDoor.Bruker (BrukerId, ForNavn, EtterNavn, Email, PassordHash, Opprettet) VALUES (1, 'Tobias', 'Hansen', 'tob@mail.com', 'hashhash', '2022-10-10 10:19:40');
+INSERT INTO NordicDoor.Bruker (BrukerId, ForNavn, EtterNavn, Email, PassordHash, Opprettet) VALUES (2, 'Siddharth', 'Dushantha', 'satan@sidd.no', 'passordhash', '2022-10-01 23:25:21');
+
```

Comment:
Kunne vi heller holdt `Create.sql` fri for eksempeldata, slik at den kan brukes i produksjon? Vi bør helle samle alt av tilfeldig data til `AddSeedData.sql`

Denne kommentaren gjelder alle changes i denne fila

Reply:
Åja, ser ut som jeg glemte å slette det fra `Create.sql`. Når vi begynte hadde jeg koden i `Create.sql` og så flytta alt til `AddSeedData.sql` men ser ut som jeg glemte å fjerne de linjene fra `Create.sql`. Skal fikse